libs/coin-modules/coin-solana/src/helpers/kind/mappers/stakeDelegate.ts

```diff
@@ -0,0 +1,7 @@
+import { OperationType } from "@ledgerhq/types-live";
+import { OperationTypeMapper } from "../type";
+
+export const mapper: OperationTypeMapper = {
+  kind: "stake.delegate",
+  operationType: "DELEGATE" as OperationType,
```

Comment:
Why do we have an `as OperationType` here while we don't just above on the `stakeCreateAccount` ?